src/engine/init.c

```diff
@@ -788,6 +788,8 @@ server_fini(bool force)
 	 * xstreams won't start shutting down until we call dss_srv_fini below.
 	 */
 	dss_srv_set_shutting_down();
+	crt_rank_abort(dss_self_rank());
+	D_INFO("dss_rank_abort() done\n");
```

Comment:
As discussed yesterday, my understanding is that crt_rank_abort(x) cancels RPC sent to rank x, while here it seems that our intention is to cancels those sent to other ranks. Would you like to ask a crt expert to review this patch, please?

The log message should say "crt_" instead of "dss_".

Reply:
I think you are right, shall use crt_rank_abort_all(), but i am not sure if passing NULL here is right or not.